libs/langchain/langchain/text_splitter.py

```diff
@@ -1387,3 +1420,12 @@ def __init__(self, **kwargs: Any) -> None:
         """Initialize a LatexTextSplitter."""
         separators = self.get_separators_for_language(Language.LATEX)
         super().__init__(separators=separators, **kwargs)
+
+
+class COBOLTextSplitter(RecursiveCharacterTextSplitter):
```

Comment:
Would you be able to remove this class -- we're trying to limit creation of new classes for languages that can be parameterized by the Language enum

Reply:
Done